docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -101,26 +101,26 @@ kubeadm on, and run:
 kubeadm init
 ```
 
-**Note:**
+**Notes:**
 
- - You need to choose a Pod Network Plugin in the next step. Depending on what
+- Please refer to the [kubeadm reference doc](/docs/admin/kubeadm/) if you want to
+read more about the flags `kubeadm init` provides. You can also specify a 
+[configuration file](/docs/admin/kubeadm/#sample-master-configuration) instead of using flags.
+- You need to choose a Pod Network Plugin in the next step. Depending on what
 third-party provider you choose, you might have to set the `--pod-network-cidr` to
 something provider-specific. The tabs below will contain a notice about what flags
 on `kubeadm init` are required.
- - This will autodetect the network interface to advertise the master on
-as the interface with the default gateway. If you want to use a different
+- The default behaviour of kubeadm is to use the network interface for the default 
```

Comment:
possible rewording, let me know if it loses your intent: 
> Unless otherwise specified, kubeadm uses the default gateway's network interface to advertise the master's IP.